CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 ### Bug fixes
 
+### Deprecated functionality
+* Deprecated `skbio.util.flatten`. Please use standard python library functionality
+described here (http://stackoverflow.com/questions/952914/making-a-flat-list-out-of-list-of-lists-in-python)
```

Comment:
Can you use the permalink instead of this URL?

On (Feb-20-15|12:43), Jai Ram Rideout wrote:
> > @@ -6,6 +6,11 @@
> >  
> >  ### Bug fixes
> >  
> > +### Deprecated functionality
> > +* Deprecated `skbio.util.flatten`. Please use standard python library functionality
> > +described here (http://stackoverflow.com/questions/952914/making-a-flat-list-out-of-list-of-lists-in-python)
> 
> Can you please format these stackoverflow links using markdown syntax? Also add the version of scikit-bio that it will be removed in.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/scikit-bio/pull/858/files#r25101349